src/docs/reference/modules/Scripting/README.md

```diff
@@ -52,3 +56,81 @@ The File scripting engine provides methods to read file contents.
+| `variables()` | TODO |
+| `parameters()` | TODO |
+
+#### Content (`OrchardCore.Contents`)
+
+| Function | Description 
+| -------- | ----------- |
+|`newContentItem(contentTypeName: String): IContent`| Creates a new instance of a ContentType (does not persist)|
+|`createContentItem(contentTypeName: String, publish: Boolean, properties: Object): IContent`| Creates and persists a new ContentItem. Conditionally publishes it. |
+|`updateContentItem(contentItem: IContent, properties: Object)`| Updates an existing content item with the properties |
+|`getUrlPrefix(path: String): String `| Prefixes the path with the Tenant prefix (if specified) |
+
+#### Layers (`OrchardCore.Layers`)
+
+| Function | Description 
+| -------- | ----------- |
+| `isHomepage(): Boolean` | Returns true if the current request Url is the current homepage |
+| `isAnonymous(): Boolean` | Returns true if there is no authenticated user on the current request |
+| `isAuthenticated(): Boolean` | Returns true if there is an authenticated user on the current request |
+| `url(url: String): Boolean` | Returns true if the current url matches the provided url. Add a `*` to the end of the url parameter to match any url that start with  |
+| `culture(name: String): Boolean` | Returns true if the current culture name or the current culture's parent name matches the `name` argument |
+
+#### Queries (`OrchardCore.Queries`)
+
+| Function | Description 
+| -------- | ----------- |
+| `executeQuery(name: String, parameters: Dictionary<string,object>): IEnumerable<object>` | Returns the result of the query. |
+
+
+#### HTTP (`OrchardCore.Workflows.Http`)
+
+| Function | Description 
+| -------- | ----------- |
+|  `httpContext(): HttpContext` | Returns the `HttpContext` which encapsulates all HTTP-specific information about an individual HTTP request. |
+| `queryString(name: String): String | Array` | Returns the entire query string (including the leading `?`) when invoked with no arguments, or the value(s) of the parameter name passed in as an argument. |
+| `responseWrite(text: String): void` | Writes the argument string directly to the HTTP response stream. |
+| `absoluteUrl(relativePath: String): String` | Returns the absolute URL for the relative path argument.  |
+| `readBody(): String` | Returns the raw HTTP request body.  |
+| `requestForm(name: String): String | Array` | Returns the value(s) of the form field name passed in as an argument. |
+| `queryStringAsJson(): JObject` | Returns the entire query string as a JSON object. Example: ` { "param1": [ "param1-value1", "param1-value2" ], "param2": [ "param2-value1", "param2-value2" ], ... }` |
+| `requestFormAsJson(): JObject` | Returns the entire request form as a JSON object. Example: ` { "field1": [ "field1-value1", "field1-value2" ], "field2": [ "field2-value1", "field2-value2" ], ... }` |
+
+
+#### Workflows (`OrchardCore.Workflows.Http`)
```

Comment:
I moved and reformatted these from the workflow module. We should probably link to these from the workflow docs page.